brave/brave-core
# Removes payment day from monthly statement

Resolves https://github.com/brave/brave-browser/issues/8960

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim grant
- open monthly statement
- make sure that you don't see pay day in AC and monthly tabs

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
